universe/Effects.cpp

```diff
@@ -1863,6 +1936,16 @@ unsigned int CreateShip::GetCheckSum() const {
     return retval;
 }
 
+std::unique_ptr<Effect> CreateShip::Clone() const {
+    auto retval = std::make_unique<CreateShip>(ValueRef::CloneUnique(m_design_name),
+                                               ValueRef::CloneUnique(m_empire_id),
+                                               ValueRef::CloneUnique(m_species_name),
+                                               ValueRef::CloneUnique(m_name),
+                                               ValueRef::CloneUnique(m_effects_to_apply_after));
+    retval->m_design_id = ValueRef::CloneUnique(m_design_id);
```

Comment:
I suppose this is OK since the base class `Effect` has no state and `CreateShip` has no other state or initialization upon construction besides these `unique_ptr`. Still a bit awkward to have the one `CloneUnique` separately. Not sure if it would be better to skip the constructor with parameter calls...